internal/alerts/ci-prod-aks-mac-weu.json

```diff
@@ -1,277 +1,413 @@
+                        "expression": "(sum(container_memory_working_set_bytes{cluster=\"/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu\", namespace=\"monitoring\", container=\"prometheus-collector\", image!=\"\"}) by (container, pod) / sum(kube_pod_container_resource_requests{cluster=\"/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu\", namespace=\"monitoring\", container=\"prometheus-collector\", resource=\"memory\"}) by (container, pod)) > 0.9",
+                        "for": "PT3M",
+                        "labels": {
+                            "cluster": "/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu"
+                        },
+                        "annotations": {
+                            "description": "Memory usage % greater than 90 for prometheus-collector containers on cluster /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu"
+                        },
+                        "severity": 4,
+                        "resolveConfiguration": {
+                            "autoResolved": true,
+                            "timeToResolve": "PT10M"
+                        },
+                        "actions": [
+                            {
+                                "ActionProperties": {
+                                    "Icm.Enabled": "True"
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "alert": "Build over build alert - CPU usage % exceeded for ama-metrics replicaset on cluster /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu",
+                        "expression": "(sum ( node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=\"ci-prod-aks-mac-weu\", namespace=\"kube-system\", container!=\"\", pod=~\"ama-metrics.*\"}) - sum ( node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=\"ci-prod-aks-mac-weu\", namespace=\"kube-system\", pod=~\"ama-metrics-node.*\"}) - sum ( node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=\"ci-prod-aks-mac-weu\", namespace=\"kube-system\", pod=~\"ama-metrics-ksm.*\"}))/ sum(kube_pod_container_resource_limits{job=\"kube-state-metrics\", cluster=\"ci-prod-aks-mac-weu\", namespace=\"kube-system\", resource=\"cpu\"}) > 0.000200",
```

Comment:
A nice-to-have would be adding the threshold we're alerting on as a threshold line in the dashboard:
<img width="1124" alt="image" src="https://user-images.githubusercontent.com/21349004/198415488-67b4be77-1b13-48ff-9417-e623c13c0a38.png">